java/flight/flight-core/src/main/java/org/apache/arrow/flight/FlightClient.java

```diff
@@ -579,6 +579,96 @@ public FlightEndpoint renewFlightEndpoint(RenewFlightEndpointRequest request, Ca
     return result;
   }
 
+  /**
+   * Request the server to extend the lifetime of a query result set.
```

Comment:
The description doesn't seem to match setSessionOptions.

Reply:
Thanks.  Fixed!